docs/providers/aws/events/kafka.md

```diff
@@ -21,6 +21,10 @@ In order to configure lambda to trigger via `kafka` events, you must provide thr
 - `topic` to consume messages from
 - `bootstrapServers` - an array of bootstrap server addresses for your Kafka cluster
 
+Optionally, you can provide the following properties:
+
+- `consumerGroupId` - the consumer group id to use for consuming messages
```

Comment:
Would be also great to add references to https://github.com/serverless/serverless/blob/main/docs/providers/aws/guide/serverless.yml.md